apps/ui/src/components/MessageVotingPower.vue

```diff
@@ -0,0 +1,53 @@
+<script setup lang="ts">
+const props = defineProps<{
+  minVotingPower?: bigint;
+  minProposalThreshold?: bigint;
+  votingPower: {
+    status: string;
+    totalVotingPower: bigint;
+  };
```

Comment:
I'd still use `VotingPowerInfo` type there, or if we don't want to require more properties than we need then do `Pick<VotingPowerInfo, 'status' | 'totalVotingPower'>`